### ISSUE TITLE
poi with koa.js

### ISSUE THREAD

**Issue by eaststrong** [egoist/poi#384]
How could I use koa.js with poi?

**Comment by mblarsen** [comment]
tl;dr - poi is not related to backend, koa is backend

---

poi is mean for quickly getting started with the front-end application. Basically building or bunding all your javascript, css and other resources.

Koa.js is a backend framework that you would typically use to serve data through an API (REST or GraphQL) and perhaps handle user authorization, etc. For this there is no build steps involved. So really nothing for poi to do.

They can live side by side in the same repo. E.g. you could have the index.js as the entry point for the poi build and then a server.js the entry point for your Koa backend.

**Comment by egoist** [comment]
Yep, if you want to work with your backend API, you may use https://poi.js.org/#/home?id=proxy-api-request
